app/server/appsmith-server/src/main/java/com/appsmith/server/configurations/SecurityConfig.java

```diff
@@ -151,6 +151,14 @@ public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
                 .csrf()
                 .disable()
                 .addFilterAt(new CSRFFilter(), SecurityWebFiltersOrder.CSRF)
+                // Default security headers configuration from
+                // https://docs.spring.io/spring-security/site/docs/5.0.x/reference/html/headers.html.
```

Comment:
```suggestion
                // https://docs.spring.io/spring-security/site/docs/5.0.x/reference/html/headers.html
```

Nit: The period at the end does not make this URL functional

Reply:
Wow, it works fine in IntelliJ! Guess I can't expect that quality everywhere. Thanks for noticing!